pillar.example

```diff
@@ -19,3 +19,17 @@ samba_sections:
     guest ok: no
     writeable: no
     printable: yes
+  sharename:
+    comment: "This is a shared directory"
+    browseable: yes
+    writeable: yes
+    valid users: @sharegroup
+    force group: sharegroup
+    create mask: 0660
+    directory mask: 2770
+
+samba_users:
+   - login: user1 
+     password: user1sambapassword
```

Comment:
Sorry to comment on this one again, are we going to confuse users with the formatting change between the rest of the file and the `samba_users` section? Nowhere else uses formatting like this.